packages/oui-pagination/src/index.spec.js

```diff
@@ -318,18 +324,36 @@ describe("ouiPagination", () => {
                         pageSizeMax: 80
                     });
 
-                let pageSizesButtons = getPageSizeButtons(element);
-                const contextController = element.scope().$ctrl;
+                // Page sizes list should not list page size above pageSizeMax (80).
+                // So, instead of displaying 4 values (see customPageSizesList = [25, 50, 100, 200]) the page size list
+                // is limited to all values lower or equal to pageSizeMax and pageSizeMax: [25, 50, 80] (length: 3).
+                const pageSizesButtons = getPageSizeButtons(element);
 
                 expect(pageSizesButtons.length).toEqual(3);
                 expect(pageSizesButtons[2].innerHTML).toEqual("80");
+            });
+
+            it("should be reinitilized", () => {
```

Comment:
When it should be reinitilized? 
should be reinitilized when  pageSizeMax changes?